package.json

```diff
@@ -24,7 +26,8 @@
     "testonly": "mocha --reporter spec --full-trace ./dist/test/tests.js",
     "coverage": "node ./node_modules/istanbul/lib/cli.js cover _mocha -- --full-trace ./dist/test/tests.js",
     "postcoverage": "remap-istanbul --input coverage/coverage.raw.json --type lcovonly --output coverage/lcov.info",
-    "prepublish": "npm run compile"
+    "start": "webpack --config webpack.config.js",
```

Comment:
`webpack.config.js` will be called by default so just put `"webpack"` in the script invocation